docs/error.md

```diff
@@ -0,0 +1,60 @@
+# Error
+
+An object representing an error that can be triggered / thrown.
+
+
+## **Overview**
+
+The errors are used to notify the user of a critical issue or log a recoverable one. An error object is consisted of three mandatory parameters: error severity, error category and an error code. The forth parameter is a mandatory object with more information about the issue.
+
+There are two severities to an error:
```

Comment:
There are two error severity levels: Critical and recoverable.

Reply:
* Critical
* Recoverable